canonical-web-and-design/ubuntu.com
# Fix up a load of urls that have www, http, or /certification

## Done

- Fix up a load of urls that have www, http, or /certification
- Removed the google analytics tracker from urls `?_ga(.*>)`
- Fixed up the UASD copy doc

## QA

- Check out this feature branch
- Read through the code and confirm changes look good

## Issue / Card

Fixes #9855
